Lib/asyncio/base_events.py

```diff
@@ -549,7 +551,7 @@ def _asyncgen_firstiter_hook(self, agen):
                     'asyncgen': agen
                 })
 
-    async def shutdown_default_executor(self):
+    async def shutdown_default_executor(self, timeout=THREAD_JOIN_TIMEOUT):
```

Comment:
@1st1 Would you prefer this:

1) In the current form, as a default parameter for `shutdown_default_executor()`

2) In another form, having the default for *timeout* being None, and passing the constant as an argument when it's called from `asyncio.run()`?

Reply:
I'd have the default set to `None` and fix `asyncio.run()` to pass `THREAD_JOIN_TIMEOUT`.  This matches the default behaviour of `thread.join()` -- i.e. wait indefinitely.

Reply:
I'll move the constant over to `runners.py` then. If we're using it only within `asynio.run()`, it wouldn't make sense to have it within `base_events.py`.